js/views/pageNavVw.js

```diff
@@ -91,17 +96,48 @@ module.exports = Backbone.View.extend({
   },
 
   handleSocketMessage: function(response) {
-    var data = JSON.parse(response.data);
-    if(data.id == this.socketNotificationID){
-      console.log(data);
-    }
-  },
+    var data = JSON.parse(response.data),
+        username,
+        avatar,
+        n;
+
+    if (data.hasOwnProperty('notification')) {
+      n = data.notification;
+      username = n.handle ? n.handle : n.guid.substring(0,10) + '...';
+      avatar = n.image_hash ? app.serverConfig.getServerBaseUrl + '/get_image?hash=' +
+        n.image_hash + '&guid=' + n.guid : 'imgs/defaultUser.png';
+
```

Comment:
There's the disconnect. There shouldn't be a "temporary" model. Model's are being used as containers to store display state for individual views and I don't think that's the intention of a Backbone Model. But... perhaps this is a philosophical discussion and in the end there's more than one way to skin a cat.